### PR TITLE
Add color rule for INI syntax

### PR DESCRIPTION
**Pull request by gosxrgxx**
Support colors for .ini files

### CODE REVIEW

**Comment by facelessuser** [comment]
Thanks for the pull, I'll try to evaluate it sometime soon.

**Comment by facelessuser** [comment]
@gosxrgxx, I think you want a pattern more like:

```
(?i)(?:\\b(?<![-#&$])(?:color\\((?!\\s*-)|(?:hsla?|(?:ok)?(?:lch|lab)|hwb|rgba?)\\()|\\b(?<![-#&$])[\\w]{3,}(?![(-])\\b|(?<![&])\\#)
```

**Comment by facelessuser** [comment]
I think that is the default pattern though, so you can just remove that pattern entry entirely.

**Comment by gosxrgxx** [comment]
To exclude color names (green, red, etc.). Perhaps you can suggest a more suitable solution.

**Comment by facelessuser** [comment]
When I'm adding such rules, I'm keeping them generic as everyone may want different things. INI files don't have innate color requirements.

**Comment by facelessuser** [comment]
If a specific file type has a specific spec in relation to colors, that is generally when we are more selective.

**Comment by gosxrgxx** [comment]
Ok. In that case, should I remove the line with the regular expression, or will you do it?

**Comment by facelessuser** [comment]
Ideally, you could make the change in this PR.

File: color_helper.sublime-settings [hunk 1]
```diff
@@ -512,6 +512,14 @@
             "scanning": ["constant.other.color"],
             "color_class": "ass_abgr",
             "color_trigger": "(?:&H|(?<=\\\\[1-4]c)|(?<=\\\\c))[0-9a-fA-F]"
+        },
+        {
+            // INI (based on https://packagecontrol.io/packages/INI)
+            "name": "INI",
+            "syntax_files": ["INI/INI"],
+            "base_scopes": ["source.ini",],
```
Comment (facelessuser) [inline color_helper.sublime-settings:520]: Please remove the dangling comma in the list

File: color_helper.sublime-settings [hunk 2]
```diff
@@ -512,6 +512,14 @@
...
+            "color_class": "css-level-4",
+            "scanning": ["meta.mapping.value.ini"],
```
Comment (facelessuser) [inline color_helper.sublime-settings:522]: Please remove the dangling comma on this line as well.

**Review by facelessuser — CHANGES_REQUESTED** [review]
Everything looks fine except for the minor requested changes.

**Review by facelessuser — APPROVED** [review]

**Comment by facelessuser** [comment]
@gir-bot lgtm

**Pull request merged**